### DISCUSSION TITLE
本地运行 data-center项目报错

### DISCUSSION THREAD

**Discussion by w990752800** [opentiny/tiny-engine#402] [Q&A]
![image](https://github.com/opentiny/tiny-engine/assets/23731304/730a5c67-b2d5-4365-ae9d-4c9996c5c33b)

**Comment by w990752800** [comment]
env配置
![image](https://github.com/opentiny/tiny-engine/assets/23731304/81e26d15-503c-4e6b-a85d-30c825c803c2)

**Comment by lu-yg** [comment]
用的是比16还高的版本吗，可以试着运行以下命令试下
`set NODE_OPTIONS=--openssl-legacy-provider`

**Reply by w990752800** [reply]
node版本 20

**Reply by lu-yg** [reply]
你可以降低你的nodejs版本到v16,或者将上面命令放到packge.json的脚本里。

**Reply by w990752800** [reply]
可以了，但是run. dev的时候又报错： 
![Uploading image.png…]()

**Reply by w990752800** [reply]
![image](https://github.com/opentiny/tiny-engine/assets/23731304/523e039e-86e2-4f3a-9628-1f4b5c0eea60)

**Reply by w990752800** [reply]
mysql 使用的 8

**Reply by lu-yg** [reply]
这是8的加密方式和5不一样导致的，你网上搜下这个报错，按那个改下就好,搜 Client does not support

**Reply by w990752800** [reply]
跑 service 项目报错： 
![image](https://github.com/opentiny/tiny-engine/assets/23731304/3efb0d4b-19e4-4731-8794-06cf9c1ea764)

**Reply by w990752800** [reply]
![image](https://github.com/opentiny/tiny-engine/assets/23731304/ae43d58a-b6a8-4383-97c2-23477924d2aa)
